### PR TITLE
Patch 17

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Changed the verbiage to talk about the New EAC vs Classic EAC and roughly what the differences are. Removed references to  Exchange Control Panel or Exchange Management Console 
In addition, added a blurb about where to find information about message and recipient limits in EXO and pointed them to the page: https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/recipients-in-exchange-online.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ ms.custom: seo-marvel-apr2020
 
 # Recipients in Exchange Online
 
-In Exchange Online, the Exchange admin center (EAC) has replaced the Exchange Control Panel (ECP) as the GUI-based administrative tool used to manage cloud-based recipients. The EAC also replaces the Exchange Management Console in Exchange Server. For more information, see [Exchange admin center in Exchange Online](../exchange-admin-center.md).
+In Exchange Online, the new Exchange admin center (EAC) has replaced the classic Exchange admin center (EAC) as the GUI-based administrative tool used to manage cloud-based recipients. The new EAC has a different look and feel than the classic EAC. In the new EAC, under **Recipients**, the **mailboxes** (User) and **shared** (mailboxes) tabs are now merged into a single tab, **Mailboxes**. On clicking the **Mailboxes** tab, you can view the shared and user mailboxes under one list view. **Migration** tab has been moved out of **Recipients**. It is available as a separate feature. For more information, see [Exchange admin center in Exchange Online](../exchange-admin-center.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/recipients-in-exchange-online.md:23]:
```suggestion
In Exchange Online, the new Exchange admin center (EAC) has replaced the classic Exchange admin center (EAC) as the GUI-based administrative tool used to manage cloud-based recipients. The new EAC has a different look and feel than the classic EAC. In the new EAC, under **Recipients**, the **mailboxes** (User) and **shared** (mailboxes) tabs are now merged into a single tab, **Mailboxes**. On clicking the **Mailboxes** tab, you can view the shared and user mailboxes under one list view. The **Migration** tab has been moved out of **Recipients**. It is available as a separate feature. For more information, see [Exchange admin center in Exchange Online](../exchange-admin-center.md).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/recipients-in-exchange-online.md:23]: Please start the second last sentence with "The Migration tab" as indicated.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**